frontend/packages/kubevirt-plugin/integration-tests/tests/models/detailView.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars, no-undef */
```

Comment:
cc @christianvogt 

Please add `frontend/packages/kubevirt-plugin/integration-tests/.eslintrc.js`

```js
module.exports = {
  root: true,
  extends: [
    'plugin:console/base',
    'plugin:console/typescript',
    'plugin:console/node,
    'plugin:console/prettier',
  ],
};
```

The `root: true` option is used to override the parent cross-package `packages/.eslintrc.js`.

Ideally, we shouldn't repeat those `/* eslint-disable ... */` comments in multiple files.

Instead, simply edit the closest `.eslintrc.js` file in hierarchy and update it like

```js
rules: {
  'no-unused-vars': 'off',
  'no-console': ['error', { allow: ['warn', 'error', 'log', 'info'] }],
},
```